pages/overview/code-of-conduct/index.page.tsx

```diff
@@ -29,7 +29,7 @@ export default function Content({
   content: any;
 }) {
   const newTitle = 'Code of Conduct';
-
+  const fileRenderType = 'tsx';
```

Comment:
We should probably redirect to the code of conduct instead of this `tsx`.

Reply:
@DarhkVoyd Do I need to redirect to the code of conduct submodule github page?

Reply:
@DarhkVoyd  Resolved this issue